github/issues.go

```diff
@@ -131,6 +132,9 @@ func (s *IssuesService) listIssues(u string, opt *IssueListOptions) ([]Issue, *R
 		return nil, nil, err
 	}
 
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeReactionsPreview)
```

Comment:
I've emailed them, they told me they've opened an internal issue to investigate. It appears to be fixed now, so nothing more needs to be done here. /cc @izuzak